<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debugger::Internal::CdbOptionsPageWidget</class>
 <widget class="QWidget" name="Debugger::Internal::CdbOptionsPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>807</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGroupBox" name="cdbPathGroupBox">
       <property name="title">
        <string extracomment="Placeholder">Startup</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QFormLayout" name="startupFormLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="additionalArgumentsLabel">
          <property name="text">
           <string>Additional &amp;arguments:</string>
          </property>
          <property name="buddy">
           <cstring>additionalArgumentsLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="additionalArgumentsLineEdit"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QCheckBox" name="consoleCheckBox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Uses CDB's native console for console applications. This overrides the setting in Environment &gt; System. The native console does not prompt on application exit. It is suitable for diagnosing cases in which the application does not start up properly in the configured console and the subsequent attach fails.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Use CDB &amp;console</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QGroupBox" name="eventGroupBox">
       <property name="title">
        <string>Break on:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="variousGroupBox">
         <property name="title">
          <string>Various</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="ignoreFirstChanceAccessViolationCheckBox">
            <property name="text">
             <string>Ignore first chance access violations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="breakCrtDbgReportCheckBox"/>
          </item>
          <item>
           <widget class="QCheckBox" name="breakpointCorrectionCheckBox">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Attempts to correct the location of a breakpoint based on file and line number should it be in a comment or in a line for which no code is generated. The correction is based on the code model.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Correct breakpoint location</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="usePythonDumper">
            <property name="text">
             <string>Use Python dumper</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
